chef_master/source/install_server_ha.rst

```diff
@@ -273,7 +276,7 @@ For each additional frontend node you wish to add to your cluster:
 
    .. code-block:: bash
 
-     	$ chef-backend-ctl gen-server-config <FE_NAME-FQDN> > chef-server.rb.<FE_NAME>
+       chef-backend-ctl gen-server-config <FE_NAME-FQDN> > chef-server.rb.<FE_NAME>
 
 #. Copy it to ``/etc/opscode`` on the new frontend node.
 
```

Comment:
Also, after these instructions is another section `Upgrading Chef Infra Server on the Frontend Machines`

This feels like it should be more part of the `upgrade_server_ha_v2.rst` documentation and also requires you to jump to another page with the standalone upgrade instructions.  Makes it very easy for users to miss this one important step that needs to be done before they upgrade the FE nodes.

Then on item `#3` there is a second link to those instructions that's not formatted correctly and results in a bad url: https://docs.chef.io/standaloneupgradeprocess/upgrade_server.html#standalone

Reply:
Fixes the link. Keeping the section, we can revisit later.